zerver/webhooks/uptimerobot/tests.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from zerver.lib.test_classes import WebhookTestCase
+
+
+class UptimeRobotHookTests(WebhookTestCase):
+    STREAM_NAME = 'uptimerobot'
+    TOPIC = "subject"
+    URL_TEMPLATE = u"/api/v1/external/uptimerobot?stream={stream}&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'uptimerobot'
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.url = self.build_webhook_url(topic=self.TOPIC)
+
+    def test_uptimerobot_monitor_down(self) -> None:
+        message = u'The monitor ZUTesting (http://7d398dfc.ngrok.io) is DOWN (HTTP 502 - Bad Gateway).'
```

Comment:
I think this would read better as just `[ZUTesting](link) is DOWN...` without "The monitor"

Reply:
I replicated the e-mail format. Hence I used  "The monitor"
![Screenshot from 2020-02-26 03-23-32](https://user-images.githubusercontent.com/13984827/75291299-f2f0a180-5847-11ea-8f74-00d570835fc2.png)

If you want, I can change it.